ucan-invocation/event-bus/source.js

```diff
@@ -0,0 +1,37 @@
+export const UCAN_STORE_EVENT_BRIDGE_SOURCE_EVENT = 'ucan_store_bucket'
+
+/**
+ * @typedef {{ detail: {key?: string, region: string, bucketName: string}}} EventBridgeEvent
+ */
+
+/**
+ * @param {import('aws-lambda').S3Event} event
+ * @param {import('@aws-sdk/client-eventbridge').EventBridge} eventBridge
+ * @param {string} eventBusName
+ */
+ export async function notifyBus(event, eventBridge, eventBusName) {
+  const entries = event.Records
```

Comment:
it help me read this is the vars where named s3Events and busEvents

Reply:
https://github.com/web3-storage/w3infra/pull/83/commits/0d55985ea1a36176f742a3bb83ac402d9dee82a4